graphql_server/render_graphiql.py

```diff
@@ -0,0 +1,306 @@
+"""Based on (express-graphql)[https://github.com/graphql/express-graphql/blob/master/src/renderGraphiQL.js] and
+(subscriptions-transport-ws)[https://github.com/apollographql/subscriptions-transport-ws]"""
+import json
+import re
+from typing import Any, Dict, Optional, Tuple
+
+from jinja2 import Environment
+from typing_extensions import TypedDict
+
+GRAPHIQL_VERSION = "1.0.3"
+
+GRAPHIQL_TEMPLATE = """<!--
+The request to this GraphQL server provided the header "Accept: text/html"
+and as a result has been presented GraphiQL - an in-browser IDE for
+exploring GraphQL.
+If you wish to receive JSON, provide the header "Accept: application/json" or
+add "&raw" to the end of the URL within a browser.
+-->
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8" />
+  <title>{{graphiql_html_title}}</title>
+  <meta name="robots" content="noindex" />
+  <meta name="referrer" content="origin" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <style>
+    body {
+      margin: 0;
+      overflow: hidden;
+    }
+    #graphiql {
+      height: 100vh;
+    }
+  </style>
+  <link href="//cdn.jsdelivr.net/npm/graphiql@{{graphiql_version}}/graphiql.css" rel="stylesheet" />
+  <script src="//cdn.jsdelivr.net/npm/promise-polyfill@8.1.3/dist/polyfill.min.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/unfetch@4.1.0/dist/unfetch.umd.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/react@16.13.1/umd/react.production.min.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/react-dom@16.13.1/umd/react-dom.production.min.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/graphiql@{{graphiql_version}}/graphiql.min.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/subscriptions-transport-ws@0.9.16/browser/client.js"></script>
+  <script src="//cdn.jsdelivr.net/npm/graphiql-subscriptions-fetcher@0.0.2/browser/client.js"></script>
```

Comment:
Based on [1.0.0 readme](https://github.com/graphql/graphiql/tree/1.0.0/packages/graphiql#cdn-bundle), looks like the `promise-polyfill` cdn isn't needed, right? @acao 

Also I noticed that the `subscription-transport-ws` package has been archived so I am not sure if there are alternatives up to date for supporting subscriptions. Do you have any idea about it?

Reply:
`promise-polyfill` has everything to do with whether you want IE support

`subscription-transport-ws` is still an active project and should be used. we removed it from our example and never got around to providing a new subscriptions example

Reply:
Oops, I linked the wrong package, I meant [graphiql-subscriptions-fetcher](https://github.com/apollographql/GraphiQL-Subscriptions-Fetcher) which still shows up on the [use it with graphiql](https://github.com/apollographql/subscriptions-transport-ws#use-it-with-graphiql) readme section of `subscription-transport-ws`.

Reply:
Also, by looking at the `subscription-transport-ws`, I am a bit worried about the latest release as it was at March 2019 and there are several PRs and issues waiting for maintainers to be answered so not sure if it is active at another branch or fork but would be good to know.